src/converter/FlameGraph.java

```diff
@@ -143,6 +143,7 @@ private String applyReplacements(String s, Object... params) {
     private void printFrame(PrintStream out, String title, Frame frame, int level, long x) {
         int type = frameType(title);
         title = stripSuffix(title);
+        title = title.replace("\\", "\\\\");
```

Comment:
Will you wrap it in `indexOf` check, like is done below? (to avoid expensive regex processing on the fast path)

Reply:
Oh, that's why the `indexOf` was there. Who knew Java actually builds a regex for each `String.replace`...

Reply:
Done

Reply:
Thanks!